src/main/java/hellos/Hellos.java

```diff
@@ -29,13 +30,17 @@ public String generateOutput() {
     return builder.toString();
   }
 
+  
+  private static String jacobSaysHello() {
+    return "Jacob says 'Salutations!'\n";
```

Comment:
This follows the format of the other greetings and appears to have passed the tests. It should be ready to go!